app/models/cart.rb

```diff
@@ -0,0 +1,15 @@
+class Cart < ActiveRecord::Base
+  belongs_to  :user
+  has_many    :order_items
```

Comment:
I was thinking of just calling the line_items. Is it okay to have Cart line_items and Order line_items?